### PR TITLE
Call paasta_dump_locally_running_services and use the weight returned by that, …

### PR DESCRIPTION
**Pull request by EvanKrall**
… instead of calculating weight here. This allows paasta to control weight.

Corresponding paasta PR: https://github.com/Yelp/paasta/pull/3450

As a side effect, this fixes the pytest tests, and gets us very close to removing paasta_tools as a python dependency. (Just need to check whether updown_service is used or not.)

### CODE REVIEW

**Comment by EvanKrall** [comment]
Hm, I'm going to need to either get paasta-tools installed in the nerve-tools itest container (but I don't think there's a publicly available debian repo with the paasta-tools package right now?), or create a fake `paasta_dump_locally_running_services` command for the itests...

File: Makefile [hunk 1]
```diff
@@ -37,3 +37,5 @@ release:
 	git show
 	@echo 'Now run `git push origin master $(RELEASE)` to release this version'
 
+test:
+	$(MAKE) -C src test
```
Comment (nemacysts) [inline Makefile:41]: nit:
```suggestion
.PHONY: test
test:
	$(MAKE) -C src test
```

**Review by nemacysts — APPROVED** [review]

File: dockerfiles/itest/itest/run_itest.sh [hunk 2]
```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
+set -vx
+set -e
+
+echo "installing paasta-tools (dependency of nerve-tools.)"
+PAASTA_VERSION=0.144.7
```
Comment (EvanKrall) [inline dockerfiles/itest/itest/run_itest.sh:6]: TODO: bump this version to something that includes https://github.com/Yelp/paasta/pull/3450

File: src/debian/control [hunk 3]
```diff
@@ -3,7 +3,7 @@ Maintainer: John Billings <billings@yelp.com>
 Build-Depends:
     dh-virtualenv,
 
-Depends: python3.7, ${shlibs:Depends}
+Depends: python3.7, paasta-tools (>= 0.144.7), ${shlibs:Depends}
```
Comment (EvanKrall) [inline src/debian/control:6]: TODO: bump this version to something that includes https://github.com/Yelp/paasta/pull/3450

**Pull request merged**